web/_js/draw.js

```diff
@@ -203,22 +208,30 @@ function initDraw(){
 
 	function exportJson(){		
 		var exportObject = {
-			 id: 0
-			,name: document.getElementById("nameField").value
-			,description: document.getElementById("descriptionField").value
-			,website: document.getElementById("websiteField").value
-			,subreddit: document.getElementById("subredditField").value
-			,center: calculateCenter(path)
-			,path: path
+			id: entryId,
+			name: document.getElementById("nameField").value,
+			description: document.getElementById("descriptionField").value,
+			website: document.getElementById("websiteField").value,
+			subreddit: document.getElementById("subredditField").value,
+			center: calculateCenter(path),
+			path: path,
 		};
+
+		if (startPeriodField.value === endPeriodField.value) {
+			exportObject.period = [startPeriodField.value]
+		} else if (startPeriodField.value * 1 < endPeriodField.value * 1) {
+			exportObject.period = [startPeriodField.value + "-" + endPeriodField.value]
```

Comment:
If I don't misread the code, this means the JSON contains the period number (0-n), right? It would be better to translate this to the timestamp using timeConfig for storage instead.

Reply:
Currently I use the availability on the images. Using timestamp would be too fine for later atlas edits.